app/views/posts/_form.html.erb

```diff
@@ -19,7 +19,8 @@
     <% end %>
 
     <%= govuk_one_third do %>
-      <%= f.file_field :images %>
+      <%= f.file_field :images,
+        multiple: true, accept: 'image/png,image/jpeg' %>
```

Comment:
gifs too?

Reply:
Would be ideal to support mp4s too, and convert gifs to mp4, but gifs for now is :+1: